server/config/environment.go

```diff
@@ -50,8 +54,9 @@ func GetEnvironment() envVars {
 	environment.ENGINE_RETRY_WAIT = 1800  // 30 minutes wait for a step to be restarted
 	environment.ENGINE_MAX_RUNTIME = 7200 // 2 hours max run time for everything (including restarts)
 	environment.EXECUTION_MAX_RETRY = 10  // 10 executions in total allowed
-	environment.DB_PATH = "/installerstore/installer.db"
-	environment.TEMPLATE_PATH = "/installerstore/templates"
+	environment.BASE_PATH = "/installerstore"
+	environment.DB_REL_PATH = "/installer.db"    // on top of BASE_PATH
```

Comment:
The `BASE_PATH` is absolute so it is expected that it starts with `/`. However, the other paths  are relative so there shouldn't be a leading slash. 
Then, where the actual path needs to be constructed, it can be done with `filepath.Join(BASE_PATH, DB_REL_PATH)`.

Reply:
Much better, fixed.